Automattic/sensei
# Fix notice style for not logged in users in learning mode

Resolves https://github.com/Automattic/sensei/issues/7242

## Proposed Changes
* Updated styles of the notice that's shown to the user on the lesson page in learning mode when the user is not logged in.

## Testing Instructions


We don't have an exact design for this notice, so we followed the other notices we have that has two buttons.

1. Create a Course with a lesson, learning mode should be enabled for the lesson
3. Try to access the lesson without logging in
4. Make sure the notice looks as expected
5. Check with all variations of course theme
6. Check with other themes
7. Check in mobile view

#### Desktop
![Screenshot 2023-11-03 at 10 43 04 PM](https://github.com/Automattic/sensei/assets/6820724/799bce84-14a1-40b5-b03c-b3ce9df8f02a)

#### Mobile
![Screenshot 2023-11-03 at 10 43 41 PM](https://github.com/Automattic/sensei/assets/6820724/847c33c8-bbac-46ef-9de1-b7128a43c938)

## Pre-Merge Checklist
